moddingway/database/notes_database.py

```diff
@@ -65,3 +66,38 @@ def list_notes(user_id: int) -> List[Note_Display]:
         rows = cursor.fetchall()
 
         return [from_db_row(row) for row in rows]
+
+
+def get_note(note_id: int) -> Note_Display:
+    conn = DatabaseConnection()
+
+    with conn.get_cursor() as cursor:
+        query = """
+        select n.noteid, n.note, n.createdby , n.lastEditedBy  
+        from notes n
+        join users u on u.userID = n.userID
+        where n.noteid = %s
+        """
+
+        params = (note_id,)
+
+        cursor.execute(query, params)
+        row = cursor.fetchone()
+
+        return from_db_row(row)
+
+
+def delete_note(note_id: int) -> bool:
+    conn = DatabaseConnection()
+
+    with conn.get_cursor() as cursor:
+        query = """
+        delete from notes n
+        where n.noteid = %s
+        """
+        params = (note_id,)
+
+        cursor.execute(query, params)
+        rows_affected = cursor.rowcount
+
+        return rows_affected > 0
```

Comment:
think it would be more clear if we just do rows_affected == 1 since only 1 deleted row is a valid operation, unless we're planning on supporting bulk deleting notes